kouncil-backend/src/main/java/com/consdata/kouncil/config/cluster/converter/ClusterConverter.java

```diff
@@ -0,0 +1,99 @@
+import com.consdata.kouncil.config.cluster.dto.SchemaRegistrySecurityConfigDto;
+import com.consdata.kouncil.model.cluster.Cluster;
+import com.consdata.kouncil.model.schemaregistry.SchemaRegistry;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public final class ClusterConverter {
+
+    public static ClusterDto convertToClusterDto(Cluster cluster) {
+        ClusterDto clusterDto = new ClusterDto();
+        clusterDto.setId(cluster.getId());
+        clusterDto.setName(cluster.getName());
+
+        clusterDto.setGlobalJmxUser(cluster.getGlobalJmxUser());
+        clusterDto.setGlobalJmxPassword(cluster.getGlobalJmxPassword());
+        clusterDto.setGlobalJmxPort(cluster.getGlobalJmxPort());
+
+        //brokers
+        setClusterBrokers(cluster, clusterDto);
+        //cluster security
+        getClusterSecurityConfigDto(cluster, clusterDto);
+        //schema registry
+        setClusterSchemaRegistry(cluster, clusterDto);
+
+
+        return clusterDto;
+    }
+
+    private static void setClusterBrokers(Cluster cluster, ClusterDto clusterDto) {
+        cluster.getBrokers().forEach(broker -> {
+            BrokerDto brokerDto = new BrokerDto();
+            brokerDto.setBootstrapServer(broker.getBootstrapServer());
+            brokerDto.setJmxUser(broker.getJmxUser());
+            brokerDto.setJmxPassword(broker.getJmxPassword());
+            brokerDto.setJmxPort(broker.getJmxPort());
+            clusterDto.getBrokers().add(brokerDto);
+        });
+    }
+
+    private static void getClusterSecurityConfigDto(Cluster cluster, ClusterDto clusterDto) {
+        ClusterSecurityConfigDto clusterSecurityConfigDto = new ClusterSecurityConfigDto();
+        clusterSecurityConfigDto.setAuthenticationMethod(cluster.getClusterSecurityConfig().getAuthenticationMethod());
+        clusterSecurityConfigDto.setSecurityProtocol(cluster.getClusterSecurityConfig().getSecurityProtocol());
+        clusterSecurityConfigDto.setSaslMechanism(cluster.getClusterSecurityConfig().getSaslMechanism());
+        clusterSecurityConfigDto.setTruststoreLocation(cluster.getClusterSecurityConfig().getTruststoreLocation());
+        clusterSecurityConfigDto.setTruststorePassword(cluster.getClusterSecurityConfig().getTruststorePassword());
+        clusterSecurityConfigDto.setKeystoreLocation(cluster.getClusterSecurityConfig().getKeystoreLocation());
+        clusterSecurityConfigDto.setKeystorePassword(cluster.getClusterSecurityConfig().getKeystorePassword());
+        clusterSecurityConfigDto.setKeyPassword(cluster.getClusterSecurityConfig().getKeyPassword());
+
+        clusterSecurityConfigDto.setUsername(cluster.getClusterSecurityConfig().getUsername());
+        clusterSecurityConfigDto.setPassword(cluster.getClusterSecurityConfig().getPassword());
+        clusterSecurityConfigDto.setAwsProfileName(cluster.getClusterSecurityConfig().getAwsProfileName());
```

Comment:
I think you can use here something like this ``BeanUtils.copyProperties(clusterSecurityConfigDto, clusterDto)``, to simplifie the code